### ISSUE TITLE
Stuck on "Attempting to connect to server"

### ISSUE THREAD

**Issue by WavyPlaysGames** [SunTheCourier/SwitchPresence-Rewritten#56]
Entered in my client id and mac address and once I click connect, that happens.

**Comment by RealFoxass** [comment]
There is a sysmodule folder you also have to download and add, I was having the same issue for hours until I figured that out

**Comment by c6maks** [comment]
> There is a sysmodule folder you also have to download and add, I was having the same issue for hours until I figured that out

where can I download that?

**Comment by RealFoxass** [comment]
> > There is a sysmodule folder you also have to download and add, I was having the same issue for hours until I figured that out
> 
> where can I download that?

Same place you download the .nro files

**Comment by c6maks** [comment]
Yes, I have the manager applet, sysmodule, and client. Am I not missing anything?

**Comment by c6maks** [comment]
> > > There is a sysmodule folder you also have to download and add, I was having the same issue for hours until I figured that out
> > 
> > 
> > where can I download that?
> 
> Same place you download the .nro files

Right, I have the manager applet, sysmodule, and client. Am I not missing anything?
